src/Data/Internal/Record.purs

```diff
@@ -0,0 +1,14 @@
+module Data.Internal.Record where
```

Comment:
`Data.Internal` is a bit of a weird namespace - perhaps it could be `Unsafe.Record` like we have `Unsafe.Coerce` and `Unsafe.Reference` (`unsafeRefEq`).

Reply:
@LiamGoodacre mentioned potentially just merging `Data.Record.Unsafe` from `purescript-record` in here?

Reply:
Sure, that works.

Reply:
which I guess also clears up the comment below, too :D Will sort this out (and remove `focus` from all the classes - thanks @MonoidMusician) at lunch :+1: